tests/test_main.py

```diff
@@ -6,26 +6,28 @@
 from unittest.mock import patch
 
 from devenv import main
+from devenv.lib import config
+from devenv.lib.config import DEFAULT_CONFIG
 from tests.utils import chdir
 
 
-def test(tmp_path: str) -> None:
-    with patch("devenv.main.CI", True), patch(
+def _test(tmp_path: str) -> None:
+    with patch("devenv.constants.CI", True), patch(
         "devenv.bootstrap.main"
     ) as mock_bootstrap, patch("devenv.sync.main") as mock_sync:
         config_path = f"{tmp_path}/.config/sentry-devenv/config.ini"
         coderoot = f"{tmp_path}/code"
-        main.DEFAULT_CONFIG["devenv"]["coderoot"] = coderoot
+        DEFAULT_CONFIG["devenv"]["coderoot"] = coderoot
 
-        main.devenv(("/path/to/argv0", "bootstrap"), config_path)
+        config.initialize_config(config_path, DEFAULT_CONFIG)
 
-        assert mock_bootstrap.mock_calls == [call(coderoot, [])]
+        main.devenv(("/path/to/argv0", "bootstrap"), config_path)
```

Comment:
hmmm needing to pass `argv0` is weird here -- argparse's default mode ignores the first argument already so we can probably fix this up (looks like it'd just be dropping all the `argv0`s in tests and removing a `[1:]` -- but that's for a separate patch!)

Reply:
Yeah, this was a before-times thing.